backend/src/game/cards/card.py

```diff
@@ -0,0 +1,35 @@
+from backend.src.game.cards.card_type import CardType
+
+
+def set_card_type(card_type):
+    if str.lower(card_type) == "keep":
+        return CardType.keep
+    elif str.lower(card_type) == "discard":
+        return CardType.discard
+    else:
+        return CardType.invalid
+
+
+class Card(object):
+    def __init__(self, name, cost, card_type, effect, footnote):
+        self.name = name
+        self.cost = cost
+        self.effect = effect
+        self.footnote = footnote
+        self.card_type = set_card_type(card_type)
+
+    def card_details_str(self):
+        return "\nname: {0}\ncost: {1}\ntype: {2}\neffect: {3}\nfootnote: {4}".format(self.name, self.cost,
+                                                                                      self.card_type,
+                                                                                      self.effect, self.footnote)
+
+    def is_valid(self):
```

Comment:
Do we need this? Isn't validation taken care of in the parser?

Reply:
The parser uses this to check if the card is valid before appending the deck it's creating.